build_docs

```diff
@@ -187,9 +189,9 @@ def run_build_docs(args):
             mount_docs_repo_and_dockerify_path(resource_dir, resource_dir)
         arg = args.next_arg()
 
-    if not saw_out:
+    if saw_doc and not saw_out:
```

Comment:
What happens if `--out` is specified, and `--doc` is not?

Reply:
Then we'll mount the directory and do nothing with it. `--out` is only valid with `--doc` but `build_docs.pl` doesn't check. We certainly could if wanted to though.

Reply:
Could you add that? Whether it be by misunderstanding or just "derp I forgot to type that one in" it's kinder to the user that if they are truly not valid together then we should tell people instead of behaving-weirdly-even-if-it-kinda-works.

Reply:
Done!